azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -121,7 +121,7 @@ def create_extension_cgroups(self, name):
             def __impl():
                 return self._cgroups_api.create_extension_cgroups(name)
 
-            self._invoke_cgroup_operation(__impl, "Failed to create a cgroup for extension '{0}'; resource usage will not be tracked".format(name))
+            return self._invoke_cgroup_operation(__impl, "Failed to create a cgroup for extension '{0}'; resource usage will not be tracked".format(name))
```

Comment:
This was a bug in the [previous impl](https://github.com/Azure/WALinuxAgent/blob/cgroups/azurelinuxagent/ga/exthandlers.py#L871) which was not returning cgroups when created.

```
self.cgroups = CGroupConfigurator.get_instance().
create_extension_cgroups(self.get_full_name())
```

Reply:
create_agent_cgroups has the same issue, could you add a return there as well?